src/lib/services-manager/widgets/services_table.rb

```diff
@@ -217,19 +241,53 @@ def name_value(service_name)
 
       # Value for the start_mode column of a service
       #
+      # @note The value contains a special mark when it has been edited by the user,
+      #   see {#highlight_value}.
+      #
       # @param service_name [String]
       # @return [String]
       def start_mode_value(service_name)
-        ServicesManagerService.start_mode_to_human_for(service_name)
+        value = ServicesManagerService.start_mode_to_human_for(service_name)
+
+        value = highlight_value(value) if service(service_name).changed?(:start_mode)
+
+        value
       end
 
       # Value for the state column of a service
       #
+      # By default it shows the current service status, but if the user starts or stops
+      # the service, then it shows the fixed text "Active" or "Inactive", see {#current_state_value}
+      # and {#future_state_value}.
+      #
+      # @note The value contains a special mark when it has been edited by the user,
+      #   see {#highlight_value}.
+      #
       # @param service_name [String]
       # @return [String]
       def state_value(service_name)
-        state = TRANSLATIONS[:service_state][service_state(service_name)]
-        substate = TRANSLATIONS[:service_substate][service_substate(service_name)]
+        service = service(service_name)
+
+        return current_state_value(service) unless service.changed?(:active)
```

Comment:
NP: Matter of taste, why not 

```ruby
return future_state_value(service) if service.changed?(:active)

current_state_value(service)
```

instead?

Reply:
Well, I prefer to start returning the common case (when there is no change) and then the rest.

Reply:
I get it. I agree then.